examples/dist/standalone.js

```diff
@@ -187,10 +187,10 @@ var Select = React.createClass({
 				return;
 			}
 			var menuElem = React.findDOMNode(_this.refs.selectMenuContainer);
-			var controlElem = React.findDOMNode(_this.refs.control);
+			var wrapperElem = React.findDOMNode(_this.refs.control);
```

Comment:
I have the same issue and your solution work for me. Just a small typo issue. I think you wanted to get reference to the `_this.refs.wrapper` instead of `_this.refs.control`